Driver/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace Driver
+{
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using EventStore.Client;
+    using Shared.EventStore.EventStore;
+
+    class Program
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/stuart_ferguson1/Shared/pullRequest?prid=5774533)